std/haxe/Error.hx

```diff
@@ -0,0 +1,34 @@
+package haxe;
+
+import haxe.PosInfos;
+
+/**
+	Common class for errors.
+**/
+class Error {
+	/**
+		Error message.
+	**/
+	public var message(default, null):String;
+
+	/**
+		Position where the error was thrown. By default, this is the place where the error is constructed.
+	**/
+	public final posInfos:PosInfos;
+
+	public function new(message:String, ?posInfos:PosInfos) {
+		this.message = message;
+		this.posInfos = posInfos;
+	}
+
+	/**
+		Error description.
+	**/
+	public function toString():String {
+		return '$message at ${pos()}';
```

Comment:
How about using Haxe's normal format (i.e. `${pos()}: $message`)? It will work out of the box in VSCode \o/

Reply:
I'm not sure it's good approach for run time exceptions.
That VSCode argument only works for console apps or exceptions printed to console. 
In general you want more visibility for an error message rather than error position.
AFAIR all the languages/runtimes provide exception message first and then position/stack info.